src/main/java/hu/dpc/phee/operator/importer/RecordParser.java

```diff
@@ -201,7 +201,7 @@ public void processTask(DocumentContext json) {
             return;
         }
 
-        Long workflowInstanceKey = json.read("$.value.workflowInstanceKey");
+        Long workflowInstanceKey = json.read("$.value.processInstanceKey");
```

Comment:
Shouldn't this be process definition key and not process instance id?

Reply:
Below are the changes
`workflowKey` : `processDefinitionKey`
`workflowInstanceKey `  : `processInstanceKey `